### DISCUSSION TITLE
Process 'query_SampleName_BCFtools' requirement exceed available memory

### DISCUSSION THREAD

**Discussion by Faizal-Eeman** [uclahs-cds/pipeline-call-sSV#28] [Q&A] [answered]
@tyamaguchi-ucla 

The call-sSV pipeline fails at the 'query_SampleName_BCFtools' process with the following error when run on an F2 node,
`Execution cancelled -- Finishing pending tasks before exit
Error executing process > 'query_SampleName_BCFtools (1)'

Caused by:
  Process requirement exceed available memory -- req: 30 GB; avail: 3 GB`

The F2.config file for F2 node memory requirement has 30GB for 'query_SampleName_BCFtools' whereas F2 node only has 4GB of memory. Is it supposed to be 3Gb instead of 30GB? Take a look at the F2.config file below.

https://github.com/uclahs-cds/pipeline-call-sSV/blob/ca2f64bb5d91bdea2e9858f29b0183c184b9cbd5/pipeline/config/F2.config#L17-L20

**Comment by tnyamaguchi** [comment, marked as answer]
Yup, the previous developer put 30.GB instead of 3.GB and we didn't catch it. We can convert this into an issue. Good catch!

**Reply by Faizal-Eeman** [reply]
Cool, we can fix this as a new issue.
